### ISSUE TITLE
[Bug][OpenTX 2.0.17] No refesh after deleting a file

### ISSUE THREAD

**Issue by LapinFou** [opentx/opentx#2623]
Hi folks,

From the SD CARD 2/8 menu when I'm deleting a file, it doesn't disappear from the list. I need to move to a parent folder then going back in the folder to see that it has been effectively removed.

**Comment by projectkk2glider** [comment]
Tried to reproduce on simu, where I completed the simpgmspace.cpp function f_unlink() to actually delete the file:
```c
FRESULT f_unlink (const TCHAR* str)
{
  TRACE("Deleting file %s", str);
  unlink(str);
  return FR_OK;
}
```

I could not reproduce this issue, which is strange. Each time a file was deleted the contents of directory were re-read in displayed correctly without the deleted file. The functions that handle this were not changed in the last year or so!

**Comment by LapinFou** [comment]
I'm able to reproduce this problem from my Taranis, not from the simulator.

**Comment by projectkk2glider** [comment]
Ok, I will test this on my radio later.

**Comment by projectkk2glider** [comment]
There is also a related bug. The radio allows and tries to delete the `[..]` item.

**Comment by LapinFou** [comment]
Have you been able to reproduce the issue?

**Comment by projectkk2glider** [comment]
Ok, reproduced. The dir must have more files than fit one on screen. The deleted file is removed from list when it scrolls out of the display. The 2.1.x is also affected.

**Comment by projectkk2glider** [comment]
This one will be solved for 2.1 first and then ported back to 2.0

**Comment by projectkk2glider** [comment]
This needs to be ported back to the 2.0.x branch
